modules/virt-cluster-resource-requirements.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-cluster-resource-requirements_{context}"]
+= Additional hardware requirements for {VirtProductName}
+
+
+{VirtProductName} is an add-on to {product-title} and imposes additional overhead that you need to take into account when planning a cluster. Each cluster machine must meet the following hardware requirements in addition to the {product-title} requirements.
+
+[id="memory-overhead_{context}"]
+=== Memory overhead
+
+.Cluster memory requirements
+
+----
+Memory overhead per infrastructure node = 158 MiB
+----
+
+Additionally, the 21 {VirtProductName} containers that total 2179 MiB of RAM will require memory overhead that is spread across all infrastructure nodes.
```

Comment:
Perhaps we could just say "Additionally, a number of {VirtProductName} containers..."?

There are more than 21 containers that make up OpenShift Virtualization. There just happen to be (currently) 21 that fall into this category. And the number of containers is not as important as their memory footprint.

Reply:
@stu-gott Can we rephrase it as "Additionally, OpenShift Virtualization environment resources require a total of 2179 MiB of RAM that is spread across all infrastructure nodes."?

Reply:
Sure that works.